content/documentation/architecture.adoc

```diff
@@ -0,0 +1,118 @@
+---
+title: "Architecture"
+date: 2018-09-03T14:39:48-05:00
+draft: false
+---
+
+= Kiali architecture
+:imagesdir: /images/documentation/architecture
+
+Kiali is composed of two components: a backend running in the container
+application platform, and a user frontend. Moreover, Kiali depends on external
+services and components provided by other applications.
```

Comment:
`services and components provided by Kubernetes and Istio`

Reply:
I wanted to avoid using _Kubernetes_ or _OS_ in most of the text. So, instead of _Kubernetes_ I'll use _container application platform_.